mllib/src/main/scala/org/apache/spark/mllib/evaluation/RegressionMetrics.scala

```diff
@@ -31,13 +30,17 @@ import org.apache.spark.sql.DataFrame
  */
 @Since("1.2.0")
 class RegressionMetrics @Since("1.2.0") (
-    predictionAndObservations: RDD[(Double, Double)]) extends Logging {
+    predictionAndObservations: RDD[(Double, Double)], isUnbiased: Boolean = true)
```

Comment:
We'll also have to explain what `isUnbiased` means. Is this really the canonical term? something about regression through the origin, or not fitting an intercept, would be more consistent with terminology in Spark?